src/CMakeLists.txt

```diff
@@ -91,10 +93,11 @@ TARGET_LINK_LIBRARIES(vangers
 	${SDL2_NET_LIBRARY}
 	${FFMPEG_LIBRARIES}
 	${ZLIB_LIBRARIES}
-	${OPENGL_LIBRARIES}
+	${OPENGL_egl_LIBRARY}
 	${WIN_LIB}
 	${STEAM_LIBS}
 	xtool
 	xgraph
 	xsound
+	dl
```

Comment:
Is it really needed? We use static linkage (at least in rust part I seen staticlib crate)

Reply:
I should check this, but I remember that I get `symbol not found` on linux without `-ldl`